support/SubmitScripts/Frontera.sh

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash
+#SBATCH -J MyJobName       # Job name
+#SBATCH -o MyJobName.o%j   # Name of stdout output file
+#SBATCH -e MyJobName.e%j   # Name of stderr error file
+#SBATCH -p normal          # Queue (partition) name
+#SBATCH -N 2               # Total # of nodes
+#SBATCH -n 2               # Total # of tasks, must be number of nodes
+#SBATCH -t 00:30:00        # Run time (hh:mm:ss)
+# #SBATCH --mail-type=all  # Send email at begin and end of job
+# #SBATCH --mail-user=username@tacc.utexas.edu
+# #SBATCH -A account       # Project/Allocation name (req'd if you have more than 1)
+
+# Distributed under the MIT License.
+# See LICENSE.txt for details.
+
+# UPDATE THESE!!!
```

Comment:
hmm, not sure what failed (most likely me...), but I don't see the following request that I attempted to make earlier:
Can you insert here similar documentation as found in `SubmitScripts/Wheeler.sh`